Felienne/hedy
# Removes email validation field on sign-up

**Description**
In this PR we remove the need of the mail repeat on sign-up. As suggested in the issue this is no longer as important as before as teachers can also change a student password for example.

**Fixes**
This PR fixes #3017.

**How to test**
Verify that the repeat mail field is removed from the signup page and everything still works as expected.
